src/notifications/warning-message.js

```diff
@@ -6,6 +6,7 @@ export default {
     browserManipulationsOnRemoteBrowser:     'The screenshot and window resize functionalities are not supported in a remote browser. They can function only if the browser is running on the same machine and in the same environment as the TestCafe server.',
     screenshotNotSupportedByBrowserProvider: 'The screenshot functionality is not supported by the "{providerName}" browser provider.',
     videoNotSupportedByBrowser:              'Video recording is not supported by the "{browserAlias}" browser.',
+    videoBrowserResizing:                    'Resize action is executed when video is recording in the "{testName}" test',
```

Comment:
Need to describe why perform of the resize action during video recording can be a problem.

Reply:
@VasilyStrelyaev

Reply:
Dirk approved the following version:

The browser window was resized while TestCafe recorded a video. The video resolution cannot be adjusted during recording. As a result, the video content may appear broken. To avoid this, do not resize the browser window when TestCafe records a video.

Reply:
New version:

The browser window was resized while TestCafe recorded a video. TestCafe cannot adjust the video resolution during recording. As a result, the video content may appear broken. Do not resize the browser window when TestCafe records a video.